File: src/main/java/com/chenyuantech/eps/bean/Ddlog.java
package com.chenyuantech.eps.bean;

import java.io.Serializable;

public class Ddlog implements Serializable {
    private String ddid;

    private String requirementdd;

    private String replydd;

    private String status;

    @Override
	public String toString() {
		return "Ddlog [ddid=" + ddid + ", requirementdd=" + requirementdd + ", replydd=" + replydd + ", status="
				+ status + ", accid=" + accid + ", launchtime=" + launchtime + ", chgid=" + chgid + ", memo=" + memo
				+ ", pmn01=" + pmn01 + ", pmn02=" + pmn02 + "]";
	}

	private String accid;

    private String launchtime;

    private String chgid;

    private String memo;

    private String pmn01;

    private Integer pmn02;

    private static final long serialVersionUID = 1L;

    public String getDdid() {
        return ddid;
    }

    public void setDdid(String ddid) {
        this.ddid = ddid == null ? null : ddid.trim();
    }

    public String getRequirementdd() {
        return requirementdd;
    }

    public void setRequirementdd(String requirementdd) {
        this.requirementdd = requirementdd == null ? null : requirementdd.trim();
    }

    public String getReplydd() {
        return replydd;
    }

    public void setReplydd(String replydd) {
        this.replydd = replydd == null ? null : replydd.trim();
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status == null ? null : status.trim();
    }

    public String getAccid() {
        return accid;
    }

    public void setAccid(String accid) {
        this.accid = accid == null ? null : accid.trim();
    }

    public String getLaunchtime() {
        return launchtime;
    }

    public void setLaunchtime(String launchtime) {
        this.launchtime = launchtime == null ? null : launchtime.trim();
    }

    public String getChgid() {
        return chgid;
    }

    public void setChgid(String chgid) {
        this.chgid = chgid == null ? null : chgid.trim();
    }

    public String getMemo() {
        return memo;
    }

    public void setMemo(String memo) {
        this.memo = memo == null ? null : memo.trim();
    }

    public String getPmn01() {
        return pmn01;
    }

    public void setPmn01(String pmn01) {
        this.pmn01 = pmn01 == null ? null : pmn01.trim();
    }

    public Integer getPmn02() {
        return pmn02;
    }

    public void setPmn02(Integer pmn02) {
        this.pmn02 = pmn02;
    }
}

File: src/main/webapp/inc/initcolumn/DDLOG.js
/**
 * 初始化栏位 -------- //交期变更记录表
 */
 initcolumn=function(){	
	return
	[{
	    "field": "pmn01",
	    "title": "采购单号",
	    "width": 100
	  }, {
		    "field": "pmn02",
		    "title": "项次",
		    "width": 100
		  },
	  {
	    "field": "accid",
	    "title": "需求变更发起人",
	    "width": 100
	  },
	  {
	    "field": "replydd",
	    "title": "供应商回复交期",
	    "width": 100
	  },
	  {
	    "field": "launchtime",
	    "title": "发起时间",
	    "width": 100
	  },
	  {
	    "field": "chgid",
	    "title": "变更理由",
	    "width": 100
	  },
	  
	  {
	    "field": "ddid",
	    "title": "流水号",
	    "width": 100,
	    "hidden":true
	  },
	  {
	    "field": "status",
	    "title": "状态(0:不同意;1:同意)",
	    "width": 100
	  },
	  
	  {
	    "field": "requirementdd",
	    "title": "采购方需求交期",
	    "width": 100
	  } ,
	  {
		    "field": "memo",
		    "title": "备注",
		    "width": 100
		  }
	];

}




